src/components/PersonalFormContainer/index.tsx

```diff
@@ -31,13 +29,13 @@ const PersonalFormContainer = () => {
 
   return (
     <form onSubmit={handleSubmit(submitForm)}>
-      <PersonalForm agencyType={params.policyType} />
+      <PersonalForm agencyType={query.policyType} />
 
       <input
         type="hidden"
         aria-hidden={true}
         name="policyType"
-        defaultValue={params.policyType || formValues.policyType}
+        defaultValue={query.policyType || formValues.policyType}
```

Comment:
What's the difference between these?  Maybe add a comment?

Reply:
Between query and params or query and formValues?

Reply:
I meant `query` and `formValues`...but, while we're at it, what's the difference between `query` and `params`?

Reply:
Yeah, I can write a comment about that. 

`params` is from the context that reads the URL query params and stores it in `useParamsContext` context. This is actually not being used right now but I didn't delete it because I _think_ it can be used for the no-javascript solution. In that case, the URL will have multiple query params stored in that context and can be used.

`query` is also from the URL query param but it is stored in the app's _state_. That's the main difference. I did this because the first page would read something like `/library-card/new?policyType=simplye` but that value gets lost in the `params` context whereas the app's state keeps it. It might make more sense, now that I think about it, to move everything to the app's state and completely remove the params context (but that's a different PR since it hasn't touched the no-js part yet).

`formValues` is the internal state of the app as well but those values are coming from the submitted forms. 

For this specific case, the hidden input will take the query's `policyType` value first and if there is none, then it takes the `formValues`'s `policyType`. The default is `webApplicant` but the query can override it to be `simplye` or, in the future, some other policy type.

Reply:
Ah, ok.  Yes, a comment would be great--after that, I think the PR is good to go!